bundler-packages/resolutions-rs/benchmark/mocked-js-impl.js

```
@@ -0,0 +1,90 @@
+/**
+ * Mocked JavaScript implementation for benchmarking purposes
+ * 
+ * This is a minimal implementation that mimics the behavior of the actual moduleFactory
+ * but without any of the external dependencies or type issues.
+ */
+
+// Create a simple module factory that returns a Map
+export const mockedJsModuleFactory = (absolutePath, rawCode, remappings, libs, fao, sync) => {
+  // Ensure all arguments are the expected types
+  absolutePath = String(absolutePath);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)